.github/reviewer_config.yml

```diff
@@ -0,0 +1,7 @@
+addReviewers: true
+addAssignees: false
+
+reviewers:
+  - HartleyAHartley
+
+numberOfReviewers: 0
```

Comment:
nit: New line at end of file.

Reply:
Fixed in c586c2317f25c89065bb8c98ee373c70524ffe71